src/elm/Data/Inventory.elm

```diff
@@ -37,15 +43,33 @@ type alias Resources =
     Config.LevelDict Resource.Resource
 
 
+type alias InventoryRecord =
+    { wallet : Wallet
+    , multipliers : Multipliers.Model
+    , discounts : LimitedMultipliers.Model
+    , resources : Resources
+    }
+
+
 type Inventory
-    = Inventory
-        { wallet : Wallet
-        , multipliers : Multipliers.Model
-        , discounts : LimitedMultipliers.Model
-        , resources : Resources
+    = Inventory InventoryRecord
+
+
+initial : Inventory
+initial =
+    Inventory
+        { wallet = Wallet.initial
+        , multipliers = Multipliers.initial
+        , discounts = LimitedMultipliers.initial
+        , resources = initialResources
         }
 
 
+unInventory : Inventory -> InventoryRecord
```

Comment:
This should be removed and replaced with pattern matching i.e. `(Inventory { multipliers, resources})`